boards/nxp/mimxrt700_evk/mimxrt700_evk_mimxrt798s_cpu0.dts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_rt7xx_cpu0.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include "mimxrt700_evk-pinctrl.dtsi"
+
+
+/ {
+	model = "NXP MIMXRT700-EVK board";
+	compatible = "nxp,mimxrt798s";
+
+	aliases {
+		led0 = &green_led;
+		led1 = &blue_led;
+		sw0 = &user_button_1;
+		sw1 = &user_button_2;		
+	};
+
+	chosen {
+		zephyr,flash-controller = &MX25UW51345GXDI00;
+		zephyr,flash = &MX25UW51345GXDI00;
+		zephyr,sram = &sram0;
+		zephyr,console = &flexcomm0_lpuart0;
+		zephyr,shell-uart = &flexcomm0_lpuart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led_0 {
+			gpios = <&gpio0 18 GPIO_ACTIVE_HIGH>;
+			label = "User LED_GREEN";
+		};
+		blue_led: led_1 {
+			gpios = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+			label = "User LED_BLUE";
+		};
+	};
+
+		gpio_keys {
+		compatible = "gpio-keys";
+		user_button_1: button_0 {
+			label = "User SW5";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_0>;
+		};
+		user_button_2: button_1 {
+			label = "User SW7";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+			zephyr,code = <INPUT_KEY_1>;
+		};
+	};
+};
+
+&systick {
+	status = "okay";
+};
+
+&flexcomm0{
+	status = "okay";
+};
+
+&flexcomm0_lpuart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&pinmux_flexcomm0_lpuart>;
+	pinctrl-names = "default";
+};
+
+&flexcomm8{
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&green_led {
+	status = "okay";
+};
+
+&blue_led {
+	status = "okay";
+};
+
+&xspi0 {
+	status = "okay";
+
+	MX25UW51345GXDI00: MX25UW51345GXDI00@0 {
+		compatible = "nxp,imx-xspi-mx25um51345g";
+		/* MX25UW51345GXDI00 is 64MB, 512MBit flash part */
+		size = <DT_SIZE_M(64 * 8)>;
+		reg = <0>;
+		spi-max-frequency = <200000000>;
+		status = "okay";
+		jedec-id = [c2 81 3a];
+		erase-block-size = <4096>;
+		write-block-size = <256>;
```

Comment:
you are telling me this flash chip can only write 256 byte blocks, nothing less than that?